src/python/pants/backend/docker/tailor_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.backend.docker.tailor import PutativeDockerTargetsRequest
+from pants.backend.docker.tailor import rules as docker_tailor_rules
+from pants.backend.docker.target_types import DockerImage
+from pants.core.goals.tailor import (
+    AllOwnedSources,
+    PutativeTarget,
+    PutativeTargets,
+    PutativeTargetsSearchPaths,
+)
+from pants.engine.rules import QueryRule
+from pants.testutil.rule_runner import RuleRunner
+
+
+def test_find_putative_targets() -> None:
```

Comment:
I expect this test can grow with shifting requirements for the tailor goal for docker files.